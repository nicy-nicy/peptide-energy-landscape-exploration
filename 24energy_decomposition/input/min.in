Minimization                                                                    
&cntrl                                                                          
  imin = 1,                                                                     
  ncyc = 1,                                                                     
  maxcyc = 1,                                                                   
  igb = 8, saltcon=0.1,                                                         
  ntb    = 0,                                                                   
  cut    = 999.0,                                                               
  rgbmax = 25.0                                                                 
/
