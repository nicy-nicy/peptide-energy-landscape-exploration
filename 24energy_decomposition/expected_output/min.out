
          -------------------------------------------------------
          Amber 12 SANDER                              2012
          -------------------------------------------------------

| PMEMD implementation of SANDER, Release 12

| Run on 03/11/2023 at 23:50:12

  [-O]verwriting output

File Assignments:
|   MDIN: min.in                                                                
|  MDOUT: min.out                                                               
| INPCRD: coords.inpcrd                                                         
|   PARM: coords.prmtop                                                         
| RESTRT: min.rst                                                               
|   REFC: refc                                                                  
|  MDVEL: mdvel                                                                 
|   MDEN: mden                                                                  
|  MDCRD: mdcrd                                                                 
| MDINFO: mdinfo                                                                


 Here is the input file:

Minimization                                                                   
&cntrl                                                                         
  imin = 1,                                                                    
  ncyc = 1,                                                                    
  maxcyc = 1,                                                                  
  igb = 8, saltcon=0.1,                                                        
  ntb    = 0,                                                                  
  cut    = 999.0,                                                              
  rgbmax = 25.0                                                                
/                                                                              


 
| Conditional Compilation Defines Used:

| New format PARM file being parsed.
| Version =    1.000 Date = 03/11/23 Time = 11:18:52

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| INFO:    Reading atomic numbers from topology file.
| Duplicated    0 dihedrals

| Duplicated    0 dihedrals

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

 NATOM  =     110 NTYPES =      10 NBONH =      49 MBONA  =      64
 NTHETH =     104 MTHETA =      86 NPHIH =     228 MPHIA  =     175
 NHPARM =       0 NPARM  =       0 NNB   =     572 NRES   =       8
 NBONA  =      64 NTHETA =      86 NPHIA =     175 NUMBND =      14
 NUMANG =      26 NPTRA  =      21 NATYP =      11 NPHB   =       0
 IFBOX  =       0 NMXRS  =      21 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0

 Implicit solvent radii are ArgH and AspGluO modified Bondi2 radii (mbondi3)                                
 Replacing prmtop screening parameters with GBn2 (igb=8) values
--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

ACE                                                                             

General flags:
     imin    =       1, nmropt  =       0

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =      50, ntrx    =       1, ntwr    =     500
     iwrap   =       0, ntwx    =       0, ntwv    =       0, ntwe    =       0
     ioutfm  =       0, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       1, ntb     =       0, igb     =       8, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     = 999.00000, intdiel =   1.00000
     saltcon =   0.10000, offset  =   0.19514, surften =   0.00500
     rdt     =   0.00000, rgbmax  =  25.00000  extdiel =  78.50000
     alpb  =        0
     gbalphaH  =   0.78844, gbbetaH   =   0.79870,  gbgammaH  =    0.43733
     gbalphaC  =   0.73376, gbbetaC   =   0.50638,  gbgammaC  =    0.20584
     gbalphaN  =   0.50336, gbbetaN   =   0.31683,  gbgammaN  =    0.19292
     gbalphaOS =   0.86781, gbbetaOS  =   0.87664,  gbgammaOS =    0.38788
     gbalphaP  =   1.00000, gbbetaP   =   0.80000,  gbgammaP  =    4.85000

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       0

Energy minimization:
     maxcyc  =       1, ncyc    =       1, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010

| Intermolecular bonds treatment:
|     no_intermolecular_bonds =       1

| Energy averages sample interval:
|     ene_avg_sampling =       1

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

ACE                                                                             
 begin time read from input coords =     0.000 ps

 
 Number of triangulated 3-point waters found:        0
| Dynamic Memory, Types Used:
| Reals                3658
| Integers            10190

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------

